test/regressions/index.test.js

```diff
@@ -95,13 +101,52 @@ async function main() {
         // Move cursor offscreen to not trigger unwanted hover effects.
         page.mouse.move(0, 0);
 
-        const testcase = await page.waitForSelector(
-          '[data-testid="testcase"]:not([aria-busy="true"])',
-        );
-
         const screenshotPath = path.resolve(screenshotDir, `${route.replace(baseUrl, '.')}.png`);
         await fse.ensureDir(path.dirname(screenshotPath));
-        await testcase.screenshot({ path: screenshotPath, type: 'png' });
+
+        let testcase;
+        if (pathURL === '/stories-grid-toolbar/PrintExportSnap') {
+          // Capture the content of the print dialog. Because there is no easy way to navigate the.
+          // print window/dialog with playwrite a hacky aproach is needed.
+          this.timeout(6000);
+          let printGrid;
+          // Click the export button in the toolbar.
+          await page.$eval(`button[aria-label="Export"]`, (exportButton) => {
+            exportButton.click();
+          });
+          // Click the print export option from the export menu in the toolbar.
+          await page.$eval(`li[role="menuitem"]:last-child`, (printButton) => {
+            printButton.click();
+          });
+          // Because by default the iframe doesn't have a src the `onload` won't be fired in Chromium
```

Comment:
Outside the test environment, the print button is also not working in Chromium / Chrome. We need a better solution.

Reply:
Updated - should be fixed now - adding `src` to the `iframe` seems to fix the issue. An additional step was needed tho to remove the contents of the `body` inside the `iframe` to prevent duplicated content.